### PR TITLE
Update user agent with appversion in the middle

### PR DESCRIPTION
**Pull request by nasli**

### CODE REVIEW

**Comment by nasli** [comment]
Would you like to review @davivel ;) ?

File: Owncloud iOs Client/Branding/Customization.h [hunk 1]
```diff
@@ -130,8 +130,8 @@
 #define k_is_shown_help_link_on_login NO
 #define k_url_link_on_login @"https://owncloud.com/mobile/new"
 
-//User-Agent
-#define k_user_agent @"Mozilla/5.0 (iOS) ownCloud-iOS/"
+//User-Agent, Mozilla/5.0 (iOS) ownCloud-iOS/<appVersion> <ob_customUserAgent>
+#define k_user_agent @"Mozilla/5.0 (iOS) ownCloud-iOS/$appVersion "
```
Comment (davivel) [inline Owncloud iOs Client/Branding/Customization.h]: I would remove the whitespace from the end. Let OB add it when needed.

Comment (nasli) [inline Owncloud iOs Client/Branding/Customization.h]: 👍 Removed

**Comment by davivel** [comment]
:+1:

**Comment by jesmrec** [comment]
Approved

**Pull request merged**